plugins/commands/updateTrusted.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const baseSchema = schema.models.command.base;
+
+module.exports = () => ({
+    method: 'PUT',
+    path: '/commands/{namespace}/{name}/trusted',
+    config: {
+        description: "Update a command's trusted property",
```

Comment:
single quotes?

Reply:
eslint not complaining so i guess it’s ok with single q inside double q? unless u want me to backlash escape it?

Reply:
I don't mind but just thought it looked odd haha
So used to eslint complaining.

Reply:
haha, u should try coding in java, again, if never…